.buildkite/core.rayci.yml

```diff
@@ -251,6 +251,21 @@ steps:
     depends_on: oss-ci-base_build
     job_env: oss-ci-base_build
 
+  - label: ":ray: core: HA integration tests"
+    tags:
+      - python
+      - docker
+    instance_type: medium
+    commands:
+      - BUILDKITE_PULL_REQUEST=false bazel run //ci/ray_ci:build_in_docker -- docker --python-version 3.8 --platform cpu
```

Comment:
you can tag the image with a random tag with the `--canonical-flag` tag

`bazel run //ci/ray_ci:build_in_docker -- docker --python-version 3.8 --platform cpu --canonical-tag v1`

why do we need the BUILDKITE_PULL_REQUEST=false?

Reply:
done.

I overwritten the `BUILDKITE_PULL_REQUEST` so that it does not generate the `pr-` prefix. do not need it when using `--canonical-tag` now.